### ISSUE TITLE
🐛 Bug: Contrast is too low in dark mode

### ISSUE THREAD

**Issue by xavdid** [LearningTypeScript/site#89]
### Bug Report Checklist

- [X] I have [searched for related issues](https://github.com/LearningTypeScript/site/issues?q=is%3Aissue+label%3A%22type%3A+bug%22) and found none that matched my issue.
- [X] This is the appropriate issue form for the bug I would like to report.

### Expected

Links on the site should be more readable in dark mode

### Actual

The dark-grey-on-black is hard to read:

![](https://cdn.zappy.app/96963327c8212c366761920af275d9d0.png)

This same color is used on other text in dark mode, like in the table of contents. White, or another lighter color would do better instead.

### Additional Info

Sorry if this isn't really a "bug", but I wasn't sure where else to put it!

**Comment by xavdid** [comment]
Same goes for the active header: 

![](https://cdn.zappy.app/e5e2072776366136b17bd7bab27893d6.png)

**Comment by JoshuaKGoldberg** [comment]
Hmm, that's not good. Accepting PRs! Thanks for reporting.
